components/edit_post_modal/edit_post_modal.tsx

```diff
@@ -71,12 +124,19 @@ class EditPostModal extends React.PureComponent {
         };
 
         this.editModalBody = React.createRef();
+        this.lastChannelSwitchAt = undefined;
+
+        this.setTextboxLinksRef = null;
+        this.editbox = null;
     }
 
-    static getDerivedStateFromProps(props, state) {
+    static getDerivedStateFromProps(props: Props, state: State) {
         if (props.editingPost.show && !state.prevShowState) {
             return {
-                editText: props.editingPost.post.message_source || props.editingPost.post.message,
+                editText:
+          props.editingPost.post.message_source ||
+          props.editingPost.post.message ||
+          '',
```

Comment:
Shouldn't need this 3rd case as `props.editingPost.post.message` should always be defined.